ui/src/components/providers.tsx

```diff
@@ -34,6 +34,7 @@ const oidcConfig = {
   redirect_uri: import.meta.env.VITE_UI_URL || 'http://localhost:5173/',
   client_id: import.meta.env.VITE_CLIENT_ID || 'ort-server-ui-dev',
   automaticSilentRenew: true,
+  loadUserInfo: true,
```

Comment:
Well the access token is actually a separate issue, but I think maybe you meant the id token. But yes, this allows to take the roles away from the id token, and instead they are fetched from the userinfo endpoint. They are still accessed the same way in the useUser hook, because react-oidc-context seems to return them in the same object regardless of whether it got them from the id token or from the userinfo endpoint. I did verify from the network tab in the web developer tools in the browser that a GET request is made to the userinfo endpoint when this loadUserInfo is set to true.